### ISSUE TITLE
Optional (aka, Im unsure how good this is): Add links to outputs in reports

### ISSUE THREAD

**Issue by PatrickAlphaC** [Cyfrin/aderyn#16]
When you get a markdown report in aderyn, it lists locations like so:

```
- Found in src/TSwapPool.sol: Line: 229
- Found in src/TSwapPool.sol: Line: 359
```

It would be cool if those were also links so that they could be `CMD + click` (on mac) or `CTRL + click`ed into and developers can go directly to the effected lines, like so:

```
- Found in [src/TSwapPool.sol](src/TSwapPool.sol) Line: 229
```

The above example brings you to the effected file, would be sick to even bring it to the line #

**Comment by alexroan** [comment]
This would work if the report.md file is always output into the project root directory, rather than where the `aderyn` command was run.

What do you think about enforcing that it prints the report to the project root? I'm unsure.

**Comment by alexroan** [comment]
This issue is fixed by #73

**Comment by alexroan** [comment]
This will be released in 0.0.8
